contents/docs/libraries/js/config.mdx

```diff
@@ -31,35 +31,35 @@ There are many configuration options, most of which you do not have to ever worr
+| `api_host`<br/><br/>**Type:** String<br/>**Default:** `https://us.i.posthog.com`                                                                                                          | URL of your PostHog instance.                                                                                                                                                                                                                                                                                  |
+| `ui_host`<br/><br/>**Type:** String<br/>**Default:** undefined                                                                                                                            | If using a [reverse proxy](/docs/advanced/proxy) for `api_host` then this should be the actual PostHog app URL (e.g. `https://us.posthog.com`). This ensures that links to PostHog point to the correct host.                                                                                                  |
+| `autocapture`<br/><br/>**Type:** Boolean or AutocaptureConfig<br/>**Default:** `true`                                                                                                     | Determines if PostHog should [autocapture](/docs/product-analytics/autocapture) events. This setting does not affect capturing pageview events (see `capture_pageview`).  [See below for `AutocaptureConfig`](#configuring-autocapture)                                                                        |
+| `before_send`<br/><br/>**Type:** Function<br/>**Default:** `function () {}`                                                                                                               | A function that allows you to amend or reject events before they are sent to PostHog. [See below for more information](/docs/libraries/js/features#amending-or-sampling-events)                                                                                                                                |
+| `bootstrap`<br/><br/>**Type:** Object<br/>**Default:** `{}`                                                                                                                               | An object containing the `distinctID`, `isIdentifiedID`, and `featureFlags` keys, where `distinctID` is a string, and `featureFlags` is an object of key-value pairs                                                                                                                                           |
+| `capture_pageview`<br/><br/>**Type:** Boolean or String<br/>**Default:** `true`                                                                                                           | Determines if PostHog should automatically capture pageview events. The default is to capture using page load events. If the special string `history_change` is provided, PostHog will capture pageviews based on path changes by listening to the browser's history API which is useful for Single Page Apps. |
```

Comment:
This is the only actual change, all the others were due to the formatter adding one extra space to line up the last `|`

Reply:
lol